### DISCUSSION TITLE
My project on github pages is not showing on other devices (404 error)

### DISCUSSION THREAD

**Discussion by alar12** [community/community#78152] [New to GitHub]
### Select Topic Area

Question

### Body

I am a new user on GitHub, and I have experienced this problem:

When I share my public page (https://alar12.github.io/about), it is not visible, but I can easily access it on my laptop.

Is there anything I could’ve done wrong, with the setup?

Any input is appreciated - thanks a lot!

-Adithya.

**Comment by Kiran1689** [comment]
Hi @alar12 
If you are using BrowserRouter for routing,  try to change it to HashRouter and see if its works or not...
